backend/test/card_game/api_play_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns card-game.api-play-test
+  (:require [expectations.clojure.test :refer :all]
+            [card-game.api.base :as api]
+            [configs :as configs]))
+
+(defexpect playing-cards
+  (expect
```

Comment:
`; Cards in hand have power`

It took me a moment to read what the expectation was looking for and I'm pretty sure I wrote the test 🙃